js/ethereum-utils/src/payloads.ts

```diff
@@ -0,0 +1,143 @@
+export type HexString = `0x${string}`;
+
+export interface EcdsaSignature {
+  Ecdsa: HexString;
+}
+
+export type ChainType = 'Mainnet-Frequency' | 'Paseo-Testnet-Frequency';
```

Comment:
Do we need local or "dev" here?

Reply:
I wasn't sure we are going to need it but I'll add it.
These are going to be used for ChainId and maybe other options. Some chains have different chainIds for their mainnet and testnet and some only have one for the mainnet. I think we can use the existing pallet-revive default config for `dev` and if we don't get a new chainId for testnet then maybe we use the same for testnet also but that is a decision we can make later.